src/main/kotlin/org/ptss/support/api/dtos/responses/QuestionnaireResponse.kt

```diff
@@ -0,0 +1,10 @@
+package org.ptss.support.api.dtos.responses
+
+data class QuestionnaireResponse(
+    val id: String,
+    val title: String,
+    val description: String,
+    val estimatedTimeOfCompletion: String,
+    val assignedAt: String,
```

Comment:
why is assigned at a string? Should be date time no?

Reply:
In the swagger it is annotated this way so the response should give back a string right?

Reply:
One question, `assignedAt` is not confused with `assignedTo` right? Cause if the `assignedAt` the date at which the questionnaire was assigned, or who it was assigned to. Because if it's the date at which the questionnaire was assigned, datetime is more logical, but if it's confused with who it was assigned to, then it should be string (and maybe a more clear name, e.g. `assignedUserId` or something). 

PS: Something seems off in Swagger, always double check.